### PR TITLE
Create index file

### PR DESCRIPTION
**Pull request by Salva-perezz**
This change create a file called "index.ts" that make a console log

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("Hola");
```
Comment (Salva-perezz) [inline index.ts:1]: este console log esta mal

Comment (Salva-perezz) [reply index.ts:1]: no, esta bien

**Comment by jsalafica** [comment]
Este console.log hay que arreglarlo

File: index.ts (hunk 1 above)
Comment (alejandrocinelli) [reply index.ts:1]: :(

**Review by JuanSphereOne — APPROVED** [review]
as

**Pull request merged**